mrbgems/picoruby-machine/src/mruby/machine.c

```diff
@@ -109,6 +109,13 @@ mrb_s_stack_usage(mrb_state *mrb, mrb_value klass)
   }
 }
 
+static void
+c_Machine_mcu_name(mrb_state *mrb, mrb_value klass)
```

Comment:
```suggestion
mrb_s_mcu_name(mrb_state *mrb, mrb_value klass)
```

Plus, you have to add `mrb_define_class_method_id()` in gem_init function

Reply:
Sorry for the missing implementation.